### PR TITLE
readme-setup-changes

### PR DESCRIPTION
**Pull request by jaredatron**

### CODE REVIEW

File: learners-guild/README.md [hunk 1]
```diff
@@ -97,22 +121,24 @@ The easiest way to test changes locally (that I've found so far) is to:
...
-        $ rm packages/rocketchat-lg-slash-commands
-        $ cd ../rocketchat-lg-slash-commands
-        #
-        # BUMP VERSION IN package.js
-        #
-        $ meteor publish
-        $ cd ../Rocket.Chat
-        $ meteor add learnersguild:rocketchat-lg-slash-commands
+```bash
+meteor remove learnersguild:rocketchat-lg-slash-commands
```
Comment (jeffreywescott) [inline learners-guild/README.md]: When I wrote this, I thought it was necessary. As it turns out, it's not necessary to remove and re-add the package. If you just leave the symlinks in place, everything gets updated automatically in `.meteor/packages` and `.meteor/versions`.

File: learners-guild/README.md [hunk 2]
```diff
@@ -97,22 +121,24 @@ The easiest way to test changes locally (that I've found so far) is to:
...
+rm packages/rocketchat-lg-slash-commands
+cd ../rocketchat-lg-slash-commands
+#
+# BUMP VERSION IN package.js
+#
+meteor publish
```
Comment (jeffreywescott) [inline learners-guild/README.md]: This `meteor publish` happens from CI now, so we should remove this part.

**Comment by jeffreywescott** [comment]
Any particular reason you removed the indented version and replaced it with the triple-backquote bash stuff? Does that give extra syntax highlighting? The reason I had it the way it was originally was just for readability in the _non-browser_ mode. Not a huge deal, but ... just wondering.

File: learners-guild/README.md (hunk 1 above)
Comment (jaredatron) [inline learners-guild/README.md]: I'm just changing this from indentation to backticks. Would you like m to remove the while section?

**Comment by jaredatron** [comment]
yeah @jeffreywescott it's for syntax highlighting and (IMHO) readability lol. Both sublime, Atom and Github will syntax highlight this syntax. Cool?

File: learners-guild/README.md (hunk 2 above)
Comment (jaredatron) [inline learners-guild/README.md]: okay so just remove this one line?

File: learners-guild/README.md (hunk 1 above)
Comment (jeffreywescott) [inline learners-guild/README.md]: Yeah -- just remove it.

File: learners-guild/README.md (hunk 2 above)
Comment (jeffreywescott) [inline learners-guild/README.md]: Yep.

Comment (jaredatron) [inline learners-guild/README.md]: done

**Pull request merged**